bigframes/display/anywidget.py

```diff
@@ -65,17 +65,13 @@ class TableWidget(WIDGET_BASE):
 
     page = traitlets.Int(0).tag(sync=True)
     page_size = traitlets.Int(0).tag(sync=True)
-    row_count = traitlets.Union(
-        [traitlets.Int(), traitlets.Instance(type(None))],
-        default_value=None,
-        allow_none=True,
-    ).tag(sync=True)
+    row_count = traitlets.Int(allow_none=True, default_value=None).tag(sync=True)
     table_html = traitlets.Unicode().tag(sync=True)
     sort_column = traitlets.Unicode("").tag(sync=True)
     sort_ascending = traitlets.Bool(True).tag(sync=True)
     orderable_columns = traitlets.List(traitlets.Unicode(), []).tag(sync=True)
     _initial_load_complete = traitlets.Bool(False).tag(sync=True)
-    _batches: Optional[blocks.PandasBatches] = None
+    _batches: blocks.PandasBatches | None = None
```

Comment:
This syntax was added in Python 3.10. https://peps.python.org/pep-0604/

Currently, we are supporting Python 3.9 https://github.com/googleapis/python-bigquery-dataframes/blob/3c54c68fd86bd85784f09e7ff819d39ca6e8409b/setup.py#L155 but since this version is end-of-life, we should probably drop it sooner rather than later.

Please wait to use this syntax until we drop Python 3.9. I have filed b/470438395 to track this.